pysiaf/tests/makeup_polynomial.py

```diff
@@ -0,0 +1,14 @@
+import numpy as np
+
+def makeup_polynomial():
+    # invent a random but plausible polynomial array
+    order = 5
+    terms = (order + 1) * (order + 2) // 2
+    a = np.zeros(terms)
+    a[1] = 0.05 + 0.01 * np.random.rand(1)
```

Comment:
The use of random in this way will lead to different results every time the test is run. Either get rid of the random part, or set the random_seed so that results are repeatable.